src/mnode/mnode-governance.cpp

```diff
@@ -121,12 +121,38 @@ CAmount CMasternodeGovernance::IncrementTicketPaidAmount(CAmount payment, CGover
     return aAmountPaid;
 }
 
+bool CMasternodeGovernance::CanVote(std::string& strErrorRet) const
+{
+    if (!masterNodeCtrl.IsActiveMasterNode()){
+        strErrorRet = "Only Active Master Node can vote";
+        return false;
+    }
+
+    auto masterNode = CMasternode{};
+    if(!masterNodeCtrl.masternodeManager.Get(masterNodeCtrl.activeMasternode.outpoint, masterNode)){
+        strErrorRet = "Failure retrieving Master Node";
+        return false;
+    }
+
+    if (!masterNode.IsEnabled()){
+        strErrorRet = "Only enabled Master Node can vote";
+        return false;
+    }
+
+    constexpr static auto thirty_days_in_seconds = int32_t{30 * 24 * 60 * 60};
```

Comment:
you could just declare constexpr static int32_t (or int as the type of parameter in IsBroadcastedWithin function)